setup.cfg

```diff
@@ -65,3 +65,8 @@ strict_equality = True
 warn_redundant_casts = True
 warn_return_any = True
 warn_unreachable = True
+plugins = pydantic.mypy
```

Comment:
shouldn't we add smth like that  to dandi-schema eventually?

Reply:
This is included in https://github.com/dandi/dandi-schema/pull/100.